src/App.tsx

```diff
@@ -1,7 +1,9 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
-
 import './App.scss';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { HomePage } from './pages/HomePage';
+import { PageNotFound } from './pages/PageNotFound';
+import { PeoplePage } from './pages/PeoplePage';
+import { Navbar } from './components/Navbar';
```

Comment:
Move the `import { Navbar } from './components/Navbar';` statement to the top of the import block to maintain proper import order.